action.yml

```diff
@@ -17,13 +17,14 @@ inputs:
     default: "master"
   project:
     description: >
-      The project id (e.g., @sourcecred), must match a project id in any
-      provided project files.
+      A GitHub project alias (e.g., @sourcecred), meaning an organization or
+      full repository. This variable is only used if project-file (suggested)
+      is not provided. 
     default: "@${{ github.repository }}"
+    required: true
   project-file: 
     description: 'the path to a file containing a project config'
-    default: null
-    required: false
+    required: true
```

Comment:
Sorry I misspoke in the call - I had first thought we could remove the project alias because we just needed the file, but in fact we need both:

  - project-file refers to the json file to describe the project configuration
  - project is the alias (index) for the project represented in that file

For example, we can provide the project file to sourcecred, but then we need the project identifier (@sourcecred) to run load. The confusing thing was that previously the input script was using "project" to describe the project file.

Reply:
Load doesn't need an "alias" only the project file and optionally weights.

```
node bin/sourcecred.js load --project ${project_file} --weights ${weights_file}
```

This has everything you need in terms of arguments. Besides that it's ENV vars.
- `SOURCECRED_GITHUB_TOKEN`
- `SOURCECRED_DIRECTORY`

---

This is also an option though:

```
node bin/sourcecred.js load @orgname --weights ${weights_file}
```

What this does is to load **all Github repositories** of that user/org without using a project file. It's _not_ an alias.

If we did both:

```
node bin/sourcecred.js load @orgname --project ${project_file} --weights ${weights_file}
```

We'll load both of them:
1. The provided org/user.
2. The project file as well.

I think that was part of what caused

Reply:
@Beanow scores _does_ need the alias. I stayed away from loading a GitHub alias, because this goes back to when GitHub was the only player in town, and this should be deprecated.

Reply:
Also, I tested with this update, and the rate limit wasn't hit as it was before it.

Reply:
One more note! @wchargin has done some much needed cleaning up of this script in #4, so I'm trying to keep changes (other than the project variable) to a minimum to make it easier for him to integrate with his PR. If we need to refactor/ tweak something here, let's do it in a fresh PR after that.

Reply:
Aah got it, we need the project ID for the scores command.
I'll approve, as it does fix the load call.

To remove this extra argument from the action though. Would it make sense to parse this from the project file using `jq`? I'm ok with making this a follow-up, or to wait on a change of SourceCred's CLI to not require this step.

Reply:
This could definitely be an option! I held off on this approach because I wasn't sure if a file could include more than one project identifier. Very likely we'd encourage multiple files, but I could see a case of one small file where there are different project configurations for the same project. Anyhoo, definitely +1 let's revisit this after early work to see if we can remove it. I'll open an issue.